GoMeet/src/test/java/com/google/sps/EmailServletTest.java

```diff
@@ -0,0 +1,108 @@
+package com.google.sps;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNotNull;
+import static org.mockito.Mockito.*;
+import org.mockito.MockedStatic;
+import org.mockito.Mockito;
+import static org.mockito.Mockito.atLeast;
```

Comment:
Nit: [Style guide](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing) says to list static and non-static imports in two blocks separated by one line, with static imports first. Within the blocks, imports are ordered alphabetically.